androidapp/DansFrappesRaps/app/src/main/java/com/cs3450/dansfrappesraps/ui/screens/DetailMenuScreen.kt

```diff
@@ -0,0 +1,127 @@
+package com.cs3450.dansfrappesraps.ui.screens
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.navigation.NavController
+import com.cs3450.dansfrappesraps.ui.components.IngredientItem
+import com.cs3450.dansfrappesraps.ui.components.Loader
+import com.cs3450.dansfrappesraps.ui.viewmodels.EditMenuViewModel
+import kotlinx.coroutines.async
+import kotlinx.coroutines.delay
+
+@Composable
+fun DetailMenuScreen(navController: NavController, id: String?) {
+    var viewModel: EditMenuViewModel = viewModel()
```

Comment:
This can be a different ticket if needed, but we will need to create a different ViewModel for this screen, it will have some of the same methods but will also have pieces for adding favorites and creating an order item and potentially temporary drinks + displaying favorited drinks